### DISCUSSION TITLE
Real-time collaboration

### DISCUSSION THREAD

**Discussion by LifeCanvasDev** [godotengine/godot-proposals#4480] [Editor]
I would like to see real-time collaboration in Godot, as is already possible in Unreal Engine (the UE implementation of real-time collab is really amazing and makes team work on a game project extremely easy, efficient and convenient, with minimal effort!)

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/851.

**Reply by LifeCanvasDev** [reply]
Thanks - it is great news that more Godot users see the value of such a feature :-)
